### PR TITLE
Redesign

### PR DESCRIPTION
**Pull request by skliarzfort**
## PR Description

TBD

## Checklist
- [x] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: css/bootstrap.min.css [hunk 1]
```diff
@@ -4920,7 +4920,7 @@ textarea.input-group-sm>.input-group-btn>.btn {
 }
 
 .navbar-static-top {
-    z-index: 1000;
+    z-index: 10;
```
Comment (tolerants) [inline css/bootstrap.min.css:4923]: Why do you change the bootstrap source code?

**Review by tolerants — CHANGES_REQUESTED** [review]
Please configure your code editor to have the same tab size everywhere (4 spaces) and to have a new line at the end of each file.

File: scss/components/_button.scss [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+		background: #eaedf0;
+		color: #767c81;
+		border-color: #dfe2e6;
+	}
+
+	[class*="icon-"] {
+		margin: 0 .7em 0 0;
+	}
+
+	&.small {
```
Comment (tolerants) [inline scss/components/_button.scss:30]: Please use a modifier.

File: scss/components/_button.scss [hunk 3]
```diff
@@ -0,0 +1,35 @@
+.button {
+	display: inline-flex;
+	vertical-align: top;
+	align-items: center;
+	justify-content: center;
+	border: 2px solid #dfe2e6;
```
Comment (tolerants) [inline scss/components/_button.scss:6]: Please move all colors into variables. Please check it everywhere.

File: scss/components/_header.scss [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+	z-index: 100;
+	&-inner {
+		display: flex;
+		justify-content: space-between;
+	}
+
+	.container {
+		&:before,
+		&:after {
+			content: none;
```
Comment (tolerants) [inline scss/components/_header.scss:13]: But where content for `container` is defined? Can't find it.

File: scss/components/_footer.scss [hunk 5]
```diff
@@ -0,0 +1,186 @@
...
+
+		&[href^="http://"]:after,
+		&[href^="https://"]:after {
+			content: none;
+		}
+	}
+
+	&-bottom {
+		background: #f5f5f5;
+		color: rgb(174, 178, 182);
```
Comment (tolerants) [inline scss/components/_footer.scss:21]: Please use HEX colors.

**Review by tolerants — CHANGES_REQUESTED** [review]

**Review by tolerants — CHANGES_REQUESTED** [review]
Do you really need a 45Kb picture for a dropdown background `images/search-section-bg.jpg` (Optional)?

File: _layouts/home.html [hunk 6]
```diff
@@ -0,0 +1,148 @@
+<!DOCTYPE html>
```
Comment (tolerants) [inline _layouts/home.html:1]: Please use extension from the `default` layout.

File: package.json [hunk 7]
```diff
@@ -0,0 +1,27 @@
+{
+	"name": "",
```
Comment (tolerants) [inline package.json:2]: The same - 8 spaces indention instead of 4.

File: scss/components/_button.scss [hunk 8]
```diff
@@ -0,0 +1,44 @@
...
+    min-width: 130px;
+    transition: color .3 linear, background .3 linear, border-color .3 linear;
+
+    &:hover,
+    &:active,
+    &:focus {
+        text-decoration: none;
+    }
+
+    &_default {
```
Comment (tolerants) [inline scss/components/_button.scss:21]: Why if this is default state we need a separate class for it?

File: scss/components/_button.scss [hunk 9]
```diff
@@ -0,0 +1,44 @@
...
+            color: #767c81;
+            border-color: #dfe2e6;
+        }
+    }
+
+    [class*="icon-"] {
+        margin: 0 .7em 0 0;
+    }
+
+    &_small {
```
Comment (tolerants) [inline scss/components/_button.scss:39]: Looks like a modifier, so suggest to change to `&-small`.

File: _includes/topnav.html [hunk 10]
```diff
@@ -1,4 +1,106 @@
 <!-- Navigation -->
+<header class="main-header">
+    <div class="container">
+        <div class="main-header__inner">
+            <div class="main-header__left">
+                <strong class="main-header__logo">
+                    <a href="/">
+                        <img src="{{ "/images/logo.svg" }}" alt="Spryker Documentation" class="main-header-logo">
```
Comment (tolerants) [inline _includes/topnav.html:8]: Why not just `logo` for example? Also we are having `main-header__logo` and `main-header-logo` which is actually confusing a bit.

File: scss/components/_cards.scss [hunk 11]
```diff
@@ -0,0 +1,107 @@
+.home-section {
```
Comment (tolerants) [inline scss/components/_cards.scss:1]: How home section is related to cards? Suggest to create separate scss for home design.

File: scss/components/_fonts.scss [hunk 12]
```diff
@@ -0,0 +1,75 @@
+@font-face {
```
Comment (tolerants) [inline scss/components/_fonts.scss:1]: Actually this file is more related to icons than to fonts, so suggest to rename it into `icons`.

File: scss/components/_footer.scss [hunk 13]
```diff
@@ -0,0 +1,191 @@
...
+            }
+        }
+    }
+
+    .copyright {
+        font-weight: bold;
+    }
+}
+
+.main-footer__top {
```
Comment (tolerants) [inline scss/components/_footer.scss:85]: Please move into `.main-footer` section. The same as all the code below.

File: scss/components/_header.scss [hunk 14]
```diff
@@ -0,0 +1,274 @@
...
+        margin: 0 24px 0 0;
+
+        img {
+            display: block;
+            width: 100%;
+            height: auto;
+        }
+    }
+
+    .button {
```
Comment (tolerants) [inline scss/components/_header.scss:41]:
```suggestion
    &__button {
```

File: scss/components/_header.scss [hunk 15]
```diff
@@ -0,0 +1,274 @@
...
+        .icon-rss {
+            font-size: 16px;
+        }
+    }
+
+    &__search {
+        position: relative;
+        z-index: 3;
+
+        & > .icon-search {
```
Comment (tolerants) [inline scss/components/_header.scss:51]: Can you avoid cascade here?

File: scss/components/_header.scss [hunk 16]
```diff
@@ -0,0 +1,274 @@
...
+        }
+    }
+}
+
+.main-nav {
+    margin: 0 24px 0 0;
+    position: relative;
+
+    &:hover {
+        & > .main-nav__drop {
```
Comment (tolerants) [inline scss/components/_header.scss:94]: Why do you need `>` here?

File: scss/components/_header.scss [hunk 17]
```diff
@@ -0,0 +1,274 @@
...
+            display: block;
+        }
+
+        & > .main-nav__opener {
```
Comment (tolerants) [inline scss/components/_header.scss:98]: Why do you need `>` here?

File: scss/components/_header.scss [hunk 18]
```diff
@@ -0,0 +1,274 @@
...
+        transition: background .2s linear, border-radius .2s linear, border-color .2s linear;
+
+        &:hover,
+        &:active,
+        &:focus {
+            text-decoration: none;
+            color: $black;
+        }
+
+        &_default {
```
Comment (tolerants) [inline scss/components/_header.scss:131]: What is `default` here?

File: scss/main.scss [hunk 19]
```diff
@@ -0,0 +1,46 @@
...
+}
+
+body {
+    background: $grey-lightest;
+    display: flex;
+    flex-direction: column;
+    font-family: $main-font;
+}
+
+.home-page {
```
Comment (tolerants) [inline scss/main.scss:20]: Suggest to move into the `pages/_home.scss`.

File: scss/main.scss [hunk 20]
```diff
@@ -0,0 +1,46 @@
...
+    .navbar {
+        display: none;
+    }
+}
+
+.page-wrapper {
```
Comment (tolerants) [inline scss/main.scss:26]: I think it can be moved into `layout.scss` the same as the code below.

File: scss/variables.scss [hunk 21]
```diff
@@ -0,0 +1,9 @@
+$black: #191919;
+$dark: #202024;
+$red: #eb553b;
+$red-dark: #c24631;
+$grey-lightest: #f5f5f5;
+$grey: #999da1;
+$headerColor: #fff;
```
Comment (tolerants) [inline scss/variables.scss:7]: Can you please explain why it's in camel case? And why white is a header color?

**Review by tolerants — CHANGES_REQUESTED** [review]

File: scss/components/_header.scss [hunk 22]
```diff
@@ -0,0 +1,274 @@
...
+            }
+        }
+
+        .algolia-autocomplete {
+            display: block !important;
+        }
+    }
+}
+
+.main-nav {
```
Comment (tolerants) [inline scss/components/_header.scss:89]: Suggest to move into separate component.

**Review by tolerants — CHANGES_REQUESTED** [review]

File: scss/components/_header.scss (hunk 17 above)
Comment (skliarzfort) [reply scss/components/_header.scss:98]: there are a few links in the multilevel menu

File: _includes/topnav.html [hunk 23]
```diff
@@ -51,12 +153,12 @@
                 {% endfor %}
                 {% endfor %}
                 {% if site.feedback_disable == null or site.feedback_disable == false %}
-			{% include feedback.html %}
-		{% endif %}
+                        {% include feedback.html %}
```
Comment (tolerants) [inline _includes/topnav.html:156]: Strange indention.

File: _layouts/default.html [hunk 24]
```diff
@@ -72,33 +72,15 @@
...
-<!-- /.container -->
-</div>
-<!-- /#main -->
+<body class="{{page.layout}}-layout">
+<div class="page-wrapper">
+    {% include topnav.html %}
+    <!-- Page Content -->
+    {{content}}
+        <!-- id="tg-sb-content" -->
     </div>
```
Comment (tolerants) [inline _layouts/default.html:81]: Please recheck closed `div`'s.

File: _layouts/page.html [hunk 25]
```diff
@@ -28,41 +24,56 @@ <h1 class="post-title-main">{{ page.title }}</h1>
...
-        {% endif %}
-        {% endfor %}
+<div class="page-content page-content--post">
+
+    <aside class="main-sidebar">
+        <div class="main-sidebar__inner">
+            <h3>Marketplace Developer Guide</h3>
+
+            {% unless page.hide_sidebar %}
+                <div class="" id="tg-sb-sidebar">
```
Comment (tolerants) [inline _layouts/page.html:34]:
```suggestion
                <div id="tg-sb-sidebar">
```

File: _layouts/page.html [hunk 26]
```diff
@@ -28,41 +24,56 @@ <h1 class="post-title-main">{{ page.title }}</h1>
...
+                    {% include sidebar.html %}
+                </div>
+            {% endunless %}
+        </div>
+    </aside>
+    <div class="post-content">
+        <div class="post-header">
+           <h1 class="post-title-main">{{ page.title }}</h1>
```
Comment (tolerants) [inline _layouts/page.html:42]: 3 spaces indention.

File: _layouts/page.html [hunk 27]
```diff
@@ -28,41 +24,56 @@ <h1 class="post-title-main">{{ page.title }}</h1>
...
+        </div>
+
+       {% if page.summary %}
+        <div class="summary">{{page.summary}}</div>
```
Comment (tolerants) [inline _layouts/page.html:46]: 1 space ;(

File: css/bootstrap.min.css (hunk 1 above)
Comment (tolerants) [reply css/bootstrap.min.css:4923]: Please remove the file as long as we are not using it anymore.

File: scss/variables.scss [hunk 28]
```diff
@@ -0,0 +1,16 @@
...
+$grey-lighten: #eaeaea;
+$grey-dark: #48494e;
+$grey-darker: #58585B;
+$grey: #999da1;
+$silver: #dfe2e6;
+$white: #fff;
+
+$main-font: 'Manrope', sans-serif;
+
+// conponent colors
```
Comment (tolerants) [inline scss/variables.scss:15]:
```suggestion
// component colors
```

File: scss/main.scss [hunk 29]
```diff
@@ -0,0 +1,17 @@
+@import 'variables';
+@import 'base/normalize';
+@import 'base/reset';
```
Comment (tolerants) [inline scss/main.scss:3]: What is the reason for using both - normalize and reset approaches?

File: scss/components/_header.scss [hunk 30]
```diff
@@ -0,0 +1,83 @@
+.main-header {
+    background: $header-bg-color;
+    padding: 15px 0;
+    z-index: 100;
+    position: relative;
+    box-shadow: 0px 6px 10px 0px rgba(0, 0, 0, 0.05),
+    0px 1px 0px 0px rgb(234, 237, 240),
```
Comment (tolerants) [inline scss/components/_header.scss:7]: There is still rgb color, please replace with HEX.

File: scss/components/_search-section.scss [hunk 31]
```diff
@@ -0,0 +1,117 @@
...
+        }
+
+        .algolia-autocomplete {
+            display: block !important;
+        }
+
+        .aa-dropdown-menu {
+            width: 100%;
+            margin: 8px 0 0;
+            background: rgb(25, 25, 25);
```
Comment (tolerants) [inline scss/components/_search-section.scss:65]: Please replace with HEX.

File: scss/components/_search-section.scss [hunk 32]
```diff
@@ -0,0 +1,117 @@
...
+            border-radius: 4px;
+            border: 1px solid rgb(88, 88, 91);
```
Comment (tolerants) [inline scss/components/_search-section.scss:67]: Please replace with HEX.

File: scss/components/_search-section.scss [hunk 33]
```diff
@@ -0,0 +1,117 @@
...
+                }
+
+                .aa-suggestion {
+                    margin: 0 0 8px;
+                }
+            }
+
+            a {
+                text-decoration: none;
+                color: rgb(153, 157, 161);
```
Comment (tolerants) [inline scss/components/_search-section.scss:105]: Please replace with HEX.

File: scss/components/_cards.scss [hunk 34]
```diff
@@ -0,0 +1,104 @@
...
+    display: flex;
+    flex-wrap: wrap;
+    max-width: 1260px;
+    margin: 0 auto;
+}
+
+.card {
+    width: calc(33.3333% - 16px);
+    margin: 0 8px 15px;
+    background: #fff;
```
Comment (tolerants) [inline scss/components/_cards.scss:11]: Please use a variable for color.

File: scss/components/_cards.scss [hunk 35]
```diff
@@ -0,0 +1,104 @@
...
+    border: 1px solid #e0e0e0;
```
Comment (tolerants) [inline scss/components/_cards.scss:12]: Please use a variable for color.

File: scss/components/_cards.scss [hunk 36]
```diff
@@ -0,0 +1,104 @@
...
+    color: #646473;
```
Comment (tolerants) [inline scss/components/_cards.scss:13]: Please use a variable for color.

File: scss/components/_cards.scss [hunk 37]
```diff
@@ -0,0 +1,104 @@
...
+
+        &-title {
+            flex: 1 1 auto;
+            font-size: 20px;
+            line-height: 28px;
+            letter-spacing: -.01em;
+            font-weight: 800;
+            padding: 0 !important;
+            margin: 0 !important;
+            color: #363639;
```
Comment (tolerants) [inline scss/components/_cards.scss:44]: Please use a variable for color.

File: scss/components/_footer.scss [hunk 38]
```diff
@@ -0,0 +1,195 @@
...
+
+        &[href^="http://"]:after,
+        &[href^="https://"]:after {
+            content: none;
+        }
+    }
+
+    &__bottom {
+        background: $grey-lightest;
+        color: #aeb2b6;
```
Comment (tolerants) [inline scss/components/_footer.scss:22]: Please use a variable for color.

File: scss/components/_footer.scss [hunk 39]
```diff
@@ -0,0 +1,195 @@
...
+            }
+        }
+    }
+
+    .copyright {
+        font-weight: bold;
+    }
+
+    &__top {
+        background: #fff;
```
Comment (tolerants) [inline scss/components/_footer.scss:86]: Please use a variable for color.

File: scss/components/_main-nav.scss [hunk 40]
```diff
@@ -0,0 +1,187 @@
...
+        &--grey {
+            border-color: $grey-lightest;
+            background: $grey-lightest;
+            font-size: 10px;
+            text-transform: uppercase;
+            .main-nav__opener-label {
+                display: none;
+            }
+            .main-nav-opener-icon {
+                color: #d3d3d4;
```
Comment (tolerants) [inline scss/components/_main-nav.scss:52]: Please use a variable for color.

File: scss/components/_main-nav.scss [hunk 41]
```diff
@@ -0,0 +1,187 @@
...
+            flex: 0 0 auto;
+            margin: 0 0 0 10px;
+            font-size: 10px;
+        }
+
+        &-label {
+            font-size: 9px;
+            line-height: 12px;
+            background: $header-bg-color;
+            color: #adafb3;
```
Comment (tolerants) [inline scss/components/_main-nav.scss:75]: Please use a variable for color.

File: scss/components/_main-nav.scss [hunk 42]
```diff
@@ -0,0 +1,187 @@
...
+        padding: 50px 0 0;
+        list-style: none;
+        font-size: 12px;
+        line-height: 16px;
+        font-weight: 600;
+        position: absolute;
+        top: 0;
+        left: 0;
+        width: 100%;
+        background: #fff;
```
Comment (tolerants) [inline scss/components/_main-nav.scss:99]: Please use a variable for color.

File: scss/components/_main-nav.scss [hunk 43]
```diff
@@ -0,0 +1,187 @@
...
+            color: $black;
+            transition: background .2s linear;
+            text-decoration: none;
+
+            &-arrow {
+                position: absolute;
+                right: 18px;
+                top: 16px;
+                font-size: 10px;
+                color: #d3d3d4;
```
Comment (tolerants) [inline scss/components/_main-nav.scss:148]: Please use a variable for color.

File: scss/components/_search-section.scss [hunk 44]
```diff
@@ -0,0 +1,117 @@
+.search-section {
+    background: #17181c url('../images/search-section-bg.jpg') no-repeat #{50% 50%/cover};
```
Comment (tolerants) [inline scss/components/_search-section.scss:2]: Please use a variable for color.

File: scss/components/_search-section.scss [hunk 45]
```diff
@@ -0,0 +1,117 @@
...
+    color: #fff;
```
Comment (tolerants) [inline scss/components/_search-section.scss:3]: Please use a variable for color.

File: scss/components/_search-section.scss [hunk 46]
```diff
@@ -0,0 +1,117 @@
...
+        margin: 0 auto;
+        position: relative;
+        position: relative;
+        z-index: 3;
+
+        &-icon {
+            position: absolute;
+            z-index: 2;
+            pointer-events: none;
+            color: #fff;
```
Comment (tolerants) [inline scss/components/_search-section.scss:26]: Please use a variable for color.

File: scss/components/_search-section.scss [hunk 47]
```diff
@@ -0,0 +1,117 @@
...
+
+        .aa-dropdown-menu {
+            width: 100%;
+            margin: 8px 0 0;
+            background: rgb(25, 25, 25);
+            border-radius: 4px;
+            border: 1px solid rgb(88, 88, 91);
+            box-shadow: 0px 20px 30px 0px rgba(0, 0, 0, 0.03),
+            0px 3px 8px 0px rgba(0, 0, 0, 0.02);
+            color: #767c81;
```
Comment (tolerants) [inline scss/components/_search-section.scss:70]: Please use a variable for color.

File: scss/components/_button.scss [hunk 48]
```diff
@@ -0,0 +1,34 @@
...
+    height: 50px;
+    padding: 0 20px;
+    min-width: 130px;
+    transition: color .3 linear, background .3 linear, border-color .3 linear;
+
+    &:hover,
+    &:active,
+    &:focus {
+        text-decoration: none;
+        background: #eaedf0;
```
Comment (tolerants) [inline scss/components/_button.scss:20]: Please use a variable for color.

File: scss/components/_button.scss [hunk 49]
```diff
@@ -0,0 +1,34 @@
...
+        color: #767c81;
```
Comment (tolerants) [inline scss/components/_button.scss:21]: Please use a variable for color.

File: gulpfile.js [hunk 50]
```diff
@@ -0,0 +1,51 @@
+var syntax         = 'scss', // Syntax: sass or scss;
+	gmWatch        = false; // ON/OFF GraphicsMagick watching "img/_src" folder (true/false). Linux install gm: sudo apt update; sudo apt install graphicsmagick
```
Comment (tolerants) [inline gulpfile.js:2]: Still 8 spaces indention.

File: scss/components/_search-section.scss [hunk 51]
```diff
@@ -0,0 +1,117 @@
...
+            outline: none;
+            outline-width: 0;
+            letter-spacing: .02em;
+            transition: color .2s linear, background .2s linear, border-color .2s linear;
+
+            // placeholder  #48494e
+            &:focus {
+                border-bottom-color: $grey-lighten;
+                color: $grey-lighten;
+
```
Comment (tolerants) [inline scss/components/_search-section.scss:54]:
```suggestion
```

**Review by tolerants — CHANGES_REQUESTED** [review]

**Review by tolerants — APPROVED** [review]

**Pull request merged**